python/setup.cfg

```diff
@@ -1,2 +1,42 @@
-# This file contains the default option values to be used during setup. An
-# example can be found at https://github.com/pypa/sampleproject/blob/master/setup.cfg
+[metadata]
```

Comment:
should this file be named pyproject.toml ?

Reply:
I am converting it to use a `pyproject.toml` right now.

Note that I reserved us the space: https://pypi.org/project/graphframes-py/